packages/eslint-config/package.json

```diff
@@ -4,13 +4,13 @@
   "main": "eslintrc.json",
   "name": "@wireapp/eslint-config",
   "peerDependencies": {
-    "eslint": ">=5",
-    "eslint-config-prettier": ">=4",
-    "eslint-plugin-jsdoc": ">=4",
-    "eslint-plugin-no-unsanitized": ">=3",
-    "eslint-plugin-prettier": ">=3",
-    "eslint-plugin-react": ">=7",
-    "prettier": ">=1.17"
+    "eslint": ">= 5",
+    "eslint-config-prettier": ">= 4",
+    "eslint-plugin-jsdoc": ">= 4",
+    "eslint-plugin-no-unsanitized": ">= 3",
+    "eslint-plugin-prettier": ">= 3",
+    "eslint-plugin-react": ">= 7",
+    "prettier": ">= 1.17"
```

Comment:
@ffflorian on a second thought.. I think I prefer without the space, usually there are no spaces here "^12.1.0", "~1.2.3", ">=1.23"

Reply:
Ah, you are right. That's also what [npm prefers](https://docs.npmjs.com/misc/semver). You can revert it.